src/hooks/Tokens.ts

```diff
@@ -31,7 +31,16 @@ function useTokensFromMap(tokenMap: TokenAddressMap, includeUserAdded: boolean):
 
     // reduce to just tokens
     const mapWithoutUrls = Object.keys(tokenMap[chainId]).reduce<{ [address: string]: Token }>((newMap, address) => {
-      newMap[address] = tokenMap[chainId][address].token
+      let token = tokenMap[chainId][address].token
+
+      if (token.equals(WETH[chainId]))
```

Comment:
I'm always hesitant about this instance comparison. I prefer checksummed address comparison. This is still sufficient for our use case now :)